.github/workflows/pr-push.yml

```
@@ -0,0 +1,35 @@
+name: CI
+on:
+  pull_request: {}
+  push:
+    branches: [main]
+jobs:
+  main:
+    name: Build, Validate and Deploy
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: w3c/spec-prod@v2
```

I suggest giving each `spec-prod` step a [name](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstepsname) so that it'll be easier to identify failures.

```suggestion
      - name: index.bs
        uses: w3c/spec-prod@v2
```